API_DOC.md

```diff
@@ -193,3 +198,76 @@ params :verified      Boolean, default: false, desc: 'Whether the posting is ver
 ```
 api :DELETE, '/housings/:id'
 ```
+
+# Volunteer
+## Index
+```
+api :GET, '/volunteer'
+param :page     Integer, desc: "What page to get"
+param :per_page Integer, default: 25 desc: "How many results per page"
+```
+### Filters
+EX: `filter[FILTER_NAME]=FILTER_VALUE`
+
+Filterable Values:
+* `:city, type: :string`
+* `:bed, type: :string`
+* `:length_of_stay, type: :string`
+* `:child_friendly, type: :boolean`
+* `:pets_accepted, type: :boolean`
+* `:verified, type: :boolean`
+* `:paid, type: :boolean`
+* `:neighborhood, type: :string`
+* `:housing_type, type: :string`
+* `:has_animals, type: :boolean`
+* `:tag, type: :scope, internal_name: :by_tag`
+* `:price, type: :string`
+## Show
+```
+api :GET, '/volunteer/:id'
+```
+## Create
+```
+api :POST, '/volunteer'
+param :skills                 Text,  required: true
+param :number_of_volunteers   Integer, required: 'organization'
+param :volunteer_notes        Text, required: 'organization'
```

Comment:
Can i ask a clarifying question? Does everything labeled with required: organization require an organization to be present? Do those represent an organization based volunteer opportunity?